.changeset/great-jobs-destroy.md

```diff
@@ -0,0 +1,6 @@
+---
+"@near-js/cookbook": minor
+"near-api-js": major
+---
+
+Update `viewFunction` on account model to only accept single object as argument
```

Comment:
Because this is a breaking change it would pay to add more detail here, including migration info. What do you think of the following:

`account.viewFunction` now takes a single object argument rather than individual args. Callsites will now need to be updated like so:
```diff
-await account.viewFunction(
-  'wrap.near',
-  'storage_balance_of',
-  { account_id: 'example.near' }
-);
+await account.viewFunction({
+  contractId: 'wrap.near',
+  methodName: 'storage_balance_of',
+  args: { account_id: 'example.near' },
+});
```